src/components/tooltip/bl-tooltip.stories.mdx

```diff
@@ -0,0 +1,141 @@
+import { html } from 'lit';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { styleMap } from 'lit/directives/style-map.js';
+import { unsafeHTML } from 'lit/directives/unsafe-html.js';
+import { Meta, Canvas, ArgsTable, Story, Preview, Source } from '@storybook/addon-docs';
+
+<Meta
+  title="Components/Tooltip"
+  component="bl-tooltip"
+  argTypes={{
+    placement: {
+        control: 'text'
+    }
+  }}
+/>
+
+export const IconTriggerTemplate = (args) => html`
+<bl-tooltip placement="${ifDefined(args.placement)}" style='--bl-tooltip-position:fixed'>
+  <bl-icon slot="tooltip-trigger" name="settings"></bl-icon>
+  Settings
+</bl-tooltip>`
+
+export const ButtonTriggerTemplate = (args) => html`
+<bl-tooltip placement="${ifDefined(args.placement)}" style='--bl-tooltip-position:fixed'>
+  <bl-button slot="tooltip-trigger" icon="plus_fill"></bl-button>
+  You can add text, photos and graphics.
+</bl-tooltip>`
+
+
+export const TextTriggerTemplate = (args) => html`
+<bl-tooltip placement="${ifDefined(args.placement)}" style='--bl-tooltip-position:fixed'>
+  <span slot="tooltip-trigger">Status</span>
```

Comment:
Text example doesn't look good styling. And @buseselvi confirmed that we can just give examples with only buttons for now.